app/controllers/nonsipp/loansmadeoroutstanding/AmountOfTheLoanController.scala

```diff
@@ -39,6 +39,7 @@ import pages.nonsipp.loansmadeoroutstanding.{AmountOfTheLoanPage, AreRepaymentsI
 import cats.{Id, Monad}
```

Comment:
Not for this line... onSubmit of OutstandingArrearsOnLoanController status “JourneyCompleted” should be saved  in mongoDb FE cache field `<entity>Progress` (e.g. [code](https://github.com/hmrc/pension-scheme-return-frontend/blob/c85083af2b430779cc87fcacc86b3c0637324bd4/app/controllers/nonsipp/employercontributions/package.scala#L53))

Reply:
This has been addressed in a code cleanup , commit needs to yet be pushed